src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const getElement = ([message, className]) => {
+  const element = document.createElement('div');
+
+  element.setAttribute('data-qa', 'notification');
+  element.innerText = message;
+  element.className = className;
+
+  return document.body.appendChild(element);
+};
+
+let leftClick = false;
+let rightClick = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    leftClick = true;
+
+    if (leftClick) {
+      resolve(['First promise was resolved', 'success']);
+    }
+  });
+
+  if (!leftClick) {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject(['First promise was rejected', 'error']);
+    }, 3000);
+  }
+});
+
+const promise2 = new Promise((resolve) => {
+  const success = (e) => {
+    e.preventDefault();
+    resolve(['Second promise was resolved', 'success']);
+  };
+
+  document.addEventListener('click', success);
+
+  document.addEventListener('contextmenu', success);
```

Comment:
In `promise2`, both 'click' and 'contextmenu' events resolve the promise. This can lead to multiple resolutions. You should ensure that the promise resolves only once by removing one of the event listeners after the promise is resolved.